NashTeamAlpha/BangazonWeb
# Added hide and show to AddToCart button and sell a product link. Adde…

## Pull Request Description

1. Changed button from "Create a product" to "Sell a product"
1. Customer cannot access sell a product or add a product to cart if they are not logged in.

## Steps to Test

1. git fetch
1. git checkout gr-SellAProductButton 
1. run the program
1. Make sure you aren't able to add to cart or sell a product unless you are logged in.

## Link to Feature Ticket

#160 
#159 
#158
